offer-manager-domain/src/main/groovy/life/qbic/business/products/create/CreateProduct.groovy

```diff
@@ -28,6 +28,7 @@ class CreateProduct implements CreateProductInput {
     @Override
     void create(Product product) {
         try {
+            //todo create new productId!!! the id is not provided from the controller. It must be done either in
```

Comment:
WHERE MUST THIS BE DONE. THE SUSPENSE IS KILLING ME 😨

Reply:
This must be done in this use case. I wanted to raise awareness that it was forgotten

Reply:
I made a suggestion so this doesn't get lost 👍